backend/db/index.js

```diff
@@ -0,0 +1,8 @@
+import Datastore from '@seald-io/nedb'
+
+export const getDataBase = () => {
+
+  const db = new Datastore({filename: 'todo_list.db', autoload: true})
+  
+  return db
+}
```

Comment:
Ideally, in backend development, a database connection is a singleton. Having a function that returns a new instance every time it is called will exhaust DB resources

Imagine you have two controllers that use this `todo_list.db`. If you call `getDataBase` in every router, you'll get two conflicting instances of `Datastore` which may corrupt the database